test/features/support/GithubAPI/Gist/DeleteGistSteps.js

```diff
@@ -7,10 +7,12 @@ const { setAuthorizationHeaders } = require('../../helpers/helpers');
 
 const urlBase = 'https://api.github.com';
 
-When('a request is used to delete a GIST from the account', async function () {
+When('a request is used to retrieve the first Gist from the account', async function () {
```

Comment:
The reason i have to use this When, is because the world resets everytime there's a change of scenario, so the url that was storaged before when the Gist was created is gone